h/hdijupyterutils/hdijupyterutils_ubi_9.3.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : hdijupyterutils
+# Version          : main
+# Source repo      : https://github.com/jupyter-incubator/sparkmagic.git
+# Tested on        : UBI:9.3
+# Language         : Python
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Aastha Sharma <aastha.sharma4@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=hdijupyterutils
+PACKAGE_VERSION=${1:-0.20.0}
+PACKAGE_URL=https://github.com/jupyter-incubator/sparkmagic.git
```

Comment:
The package name for given url(https://github.com/jupyter-incubator/sparkmagic.git) is sparkmagic. And hdijupyterutils is sub module of it. You can write build script for hdijupyterutils and not for base package

Reply:
Resolved